pkg/specs/jobs.go

```diff
@@ -29,6 +29,12 @@ import (
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/utils"
 )
 
+const (
+	// PostInitApplicationSQLRefsFolder points to the folder of
+	// postInitApplicationSQL files in the primary job with initdb.
+	PostInitApplicationSQLRefsFolder = "/etc/post-init-application-sql-refs"
```

Comment:
I think this can be private, as we use it just inside the `spec` package. I'm sorry to be so picky...

Reply:
should this go in `/etc/` ? probably we can put it in a directory inside `/controller`, I haven't check everything from scratch again so I may be forgetting sometlhing

Reply:
We already mount secrets in `/etc/superuser-secret` and `/etc/app-secret` so I don't see any downside on using `/etc` name here. Maybe we could devise a better name for that directory... I think `post-init-application-sql` it's better than `post-init-application-sql-refs`. There are no refs here but just contents.

Reply:
Yes, but those are configuration or secrets require for configuration, that's why `/etc` it's ok, but as you said this is content, that's why I think `/etc/` it's no the proper place for content file, by convention `/etc` has always contain configuration and files needed by configuration, on the other hand `/controller` contains files with all the needed by the instance manager, even file with content

Reply:
fixed with the following action:
- make the const be private
- replace `post-init-application-sql-refs` with `post-init-application-sql  `